cmd/buildkitd/config/config.go

```diff
@@ -112,6 +112,7 @@ type ContainerdConfig struct {
 	Labels    map[string]string `toml:"labels"`
 	Platforms []string          `toml:"platforms"`
 	Namespace string            `toml:"namespace"`
+	Runtime   string            `toml:"runtime"`
```

Comment:
The new option should be added to https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md as well.

Reply:
Done